javascript-carousel/main.js

```diff
@@ -0,0 +1,73 @@
+var $imagesList = document.querySelectorAll('.img');
+var $dotsDiv = document.querySelector('.dots');
+var $dotList = document.querySelectorAll('.fa-circle');
+var $leftArr = document.querySelector('.fa-chevron-left');
+var $rightArr = document.querySelector('.fa-chevron-right');
+
+function changeImg() {
+  var currIndex;
+  for (var i = 0; i < $imagesList.length; i++) {
+    if ($imagesList[i].matches('.view')) {
+      currIndex = i;
+    }
+    $imagesList[i].className = 'hidden img';
+    $dotList[i].className = 'far fa-circle';
+  }
+  currIndex++;
+  if (currIndex >= $imagesList.length) {
+    currIndex = 0;
+  }
+  $imagesList[currIndex].className = 'view img';
+  $dotList[currIndex].className = 'fas fa-circle';
+}
+
+function startCarousel() {
+  clearInterval(onLoadIntervalId);
+  var newIntervalId;
```

Comment:
If you make this variable global, I think you can avoid having a separate variable for the `id` when the page loads.